samples/bot-conversation-sso-quickstart/js/README.md

```diff
@@ -1,41 +1,92 @@
 # Bots/Messaging Extension
 
-*Bots* allow users to interact with your web service through text, interactive cards, and task modules. *Messaging extensions* allow users to interact with your web service through buttons and forms in the Microsoft Teams client. They can search, or initiate actions, in an external system from the compose message area, the command box, or directly from a message.
+Bot Framework v4 bot using Teams authentication
```

Comment:
In my mind, enable SSO need developers to config some settings on bot channel registration and AAD identity provider, I have not found the step in this tutorial.

Reply:
Just refined the ReadMe, kindly review~